pyproject.toml

```diff
@@ -31,21 +31,11 @@ dynamic = ["version"]
 dependencies = [
   # We always pin a specific version of bioio-base with
   # each release so that we avoid bioio + bioio-base misalignment.
-  "bioio-base==1.0.7",
+  "bioio-base>=1.0.7",
   "dask[array]>=2021.4.1",
-  "fsspec[http]>=2022.8.0",
```

Comment:
I think the dummy plugin reader uses this so you'll want to add it to the optional `test` deps

Reply:
never mind no they didnt. Added back in [38c66f8](https://github.com/bioio-devs/bioio/pull/126/commits/38c66f8fec21c1ae7601058c5ca7bafbd46fb38a)